lib/src/drm/fairplay/getLicenseServerUrl.ts

```diff
@@ -0,0 +1,28 @@
+import { isEmpty } from '../utils/isEmpty.js';
+
+/**
+ * Retrieves the FairPlay license server URL from InitData or DRM configuration.
+ *
+ * @param initData - The initialization data (PSSH box)
+ * @param drmConfig - The DRM configuration containing the license URL
+ * @returns The license server URL as a string
+ *
+ * @group DRM
+ * @beta
+ */
+
+export function getLicenseServerUrl(initData: Uint16Array, drmConfig: { licenseUrl?: string; }): string {
+	if (!isEmpty(drmConfig.licenseUrl) && drmConfig.licenseUrl?.startsWith('http')) {
+		return drmConfig.licenseUrl;
+	}
```

Comment:
It doesn't look like the `drmConfig` object is used anywhere else in this function besides the initial escape statement. This should be handled at the app level:

```ts
const url = drmConfig.licenseUrl || getLicenseServerUrl(initData);
```
Lets remove the second argument and early return block.

Reply:
Removed drmConfig param from the function and adjusted it accordingly including its test. This change also means that I no longer need isEmpty util, so removed that as well. 👍🏼